go/tasks/plugins/k8s/kfoperators/mpi/mpi.go

```diff
@@ -69,11 +71,19 @@ func (mpiOperatorResourceHandler) BuildResource(ctx context.Context, taskCtx plu
 	common.OverrideDefaultContainerName(taskCtx, podSpec, kubeflowv1.MPIJobDefaultContainerName)
 
 	// workersPodSpec is deepCopy of podSpec submitted by flyte
-	// WorkerPodSpec doesn't need any Argument & command. It will be trigger from launcher pod
 	workersPodSpec := podSpec.DeepCopy()
 
+	// If users don't specify "worker_spec_command" in the task config, the command/args are empty.
+	// However, in some cases, the workers need command/args.
+	// For example, in horovod tasks, each worker runs a command launching ssh daemon.
+
+	workerSpecCommand := []string{}
+	if val, ok := taskTemplateConfig["worker_spec_command"]; ok {
```

Comment:
can we make `worker_spec_command` a `const`?

Reply:
but `worker_spec_command` is mutatable. See line 82

Reply:
yes, i understand. i mean:
```
const workerSpecCommandKey = "worker_spec_command"

...
if val, ok := taskTemplateConfig[workerSpecCommandKey]; ok {
    ...
```